lib/plugins/create/templates/aws-nodejs-typescript/package.json

```diff
@@ -15,5 +15,8 @@
     "webpack": "^3.6.0"
   },
   "author": "The serverless webpack authors (https://github.com/elastic-coders/serverless-webpack)",
-  "license": "MIT"
-}
\ No newline at end of file
+  "license": "MIT",
+  "dependencies": {
```

Comment:
IMO this should be put before `"devDependencies"` (line 9).

Reply:
Now that you've merged https://github.com/serverless/serverless/pull/4732#discussion_r171312112 you can safely remove from line 22 to line 24 (`dependencies` is duplicated now) and also remove the last `,` in line 21.